templates/app/test/routes/root.test.js

```diff
@@ -1,10 +1,10 @@
 'use strict'
 
 const { test } = require('tap')
-const { build } = require('../helper')
+const { buildApplication } = require('../helper')
 
 test('default root route', async (t) => {
-  const app = build(t)
+  const app = await buildApplication(t)
```

Comment:
why do you need to await this? It's not strictly needed by fastify.

Reply:
It is not required by fastify, but by the fastify-cli logic.

for example we hit this line:

https://github.com/fastify/fastify-cli/blob/5654ea1710c5b8e67dd5e73bcfc5349a983bdb10/start.js#L89

that read the package.json and manage the ESM files.

It should not be necessary if we manage simple CJS script